cmd/tailscale/cli/up.go

```diff
@@ -768,6 +771,7 @@ func init() {
 	addPrefFlagMapping("auto-update", "AutoUpdate.Apply")
 	addPrefFlagMapping("advertise-connector", "AppConnector")
 	addPrefFlagMapping("posture-checking", "PostureChecking")
+	addPrefFlagMapping("advertise-services", "AdvertiseServices")
```

Comment:
there's no `advertise-services` flag anymore, what happens without this?

Reply:
```
$ tool/go test ./cmd/tailscale/cli    
--- FAIL: TestPrefFlagMapping (0.00s)
    cli_test.go:954: unexpected new ipn.Pref field "AdvertiseServices" is not handled by up.go (see addPrefFlagMapping and checkForAccidentalSettingReverts)
FAIL
FAIL    tailscale.com/cmd/tailscale/cli 0.019s
FAIL
```

Reply:
I think the better solution is to add a new exception in `cli_test.go` with a comment similar to the `"DriveShares"` exception.